### PR TITLE
Add eslint-plugin's skeleton for the docs

### PR DESCRIPTION
**Pull request by neriyarden**

### CODE REVIEW

**Review by unicop — APPROVED** [review]
@neriyarden great work !!

File: apps/website/docs/plugins/eslint-plugin/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+displayed_sidebar: "eslintPluginSideBar"
+---
+
+# Getting Started
+
+In 9 minutes ⏰ you will start using our configs.
+
+### Prerequisites
+
+- Prerequisites: [Node.js](https://nodejs.org/en/) (`^12.22.0`, `^14.17.0`, or `>=16.0.0`) built with SSL support. (If you are using an official Node.js distribution, SSL is always built in.)
```
Comment (unicop) [inline apps/website/docs/plugins/eslint-plugin/getting-started.md:12]: you can use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions)

like 
:::note
a note
:::

File: apps/website/docs/plugins/eslint-plugin/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Installation
+
+
+## Configuration
```
Comment (unicop) [inline apps/website/docs/plugins/eslint-plugin/getting-started.md:17]: @neriyarden add for both configuration + installation a TBD text

File: apps/website/docs/plugins/migrate-to-typescript/configs/esm-javascript.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+| **[no-commonjs-exports](../rules/no-commonjs-exports.md)**                 | ✅  |     | 🔴  |     |     |
+| **[no-commonjs-module-exports](../rules/no-commonjs-module-exports.md)**   | ✅  |     | 🔴  | 🔧  |     |
+| **[no-esm-dynamic-import](../rules/no-esm-dynamic-import.md)**             | ✅  | 🟡  |     |     |     |
+| **[no-esm-export-default](../rules/no-esm-export-default.md)**             |     |     |     |     |     |
+| **[no-require](../rules/no-require.md)**                                   | ✅  |     | 🔴  |     |     |
+| **[esm-consistent-type-import](../rules/esm-consistent-type-import.md)**   | ✅  |     | 🔴  | 🔧  |     |
+
+usage:
+
+```json
```
Comment (unicop) [inline apps/website/docs/plugins/migrate-to-typescript/configs/esm-javascript.md:21]: it is not a correct table for this module, am I right?

File: apps/website/sidebars/main-sidebars.js [hunk 4]
```diff
@@ -57,6 +57,99 @@ const sidebars = {
...
+            'plugins/eslint-plugin/configs/react-plus-jest-js',
+            'plugins/eslint-plugin/configs/react-plus-jest-js-with-ts',
+            'plugins/eslint-plugin/configs/react-plus-jest-ts',
+          ]
+        },
+        {
+          type: "category",
+          label: "Typescript-eslint",
+          link: {type: 'doc', id: 'plugins/eslint-plugin/configs/index'},
+          items: [
```
Comment (unicop) [inline apps/website/sidebars/main-sidebars.js:144]: I liked how you categorized it!

**Comment by unicop** [comment]
@neriyarden please verify that the build pass

File: apps/website/docs/plugins/eslint-plugin/configs/esm-javascript.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (neriyarden) [inline apps/website/docs/plugins/eslint-plugin/configs/esm-javascript.md:1]: @unicop have a look at this format. what do you think?

Comment (unicop) [reply apps/website/docs/plugins/eslint-plugin/configs/esm-javascript.md:1]: looks great.

**Comment by neriyarden** [comment]
@unicop the esm-javascript rules table is done

**Comment by unicop** [comment]
@neriyarden do you still have open questions? or is this ticket WIP ?

**Comment by unicop** [comment]
@neriyarden please make the build pass

**Pull request merged**